src/main/java/no/unit/nva/database/DatabaseServiceImpl.java

```diff
@@ -47,23 +46,22 @@ public DatabaseServiceImpl(DynamoDBMapper mapper) {
     @Override
     public Optional<UserDto> getUser(String username) throws InvalidUserException {
         UserDb searchObject = UserDto.newBuilder().withUsername(username).build().toUserDb();
-        DynamoDBQueryExpression<UserDb> query = createGetUserQuery(searchObject);
-        List<UserDb> result = mapper.query(UserDb.class, query);
+        DynamoDBQueryExpression<UserDb> searchUserByUsername = createGetUserQuery(searchObject);
```

Comment:
I think this is a good naming scheme, I was going to suggest "find" as a gut reaction from an English language perspective, but your naming made me realise that this is actually a search operation and as such the naming is far better as you have written it.